tensorrt_llm/_torch/pyexecutor/sampler.py

```
@@ -247,22 +250,35 @@ def _meet_max_token_stop_criteria(self, request: LlmRequest):
                                                   >= self.max_seq_len)
 
     @staticmethod
-    def _meet_stop_token_criteria(request: LlmRequest):
+    def _meet_stop_token_criteria(request: LlmRequest,tokenizer: PreTrainedTokenizerBase, new_token: Union[int, List[int], torch.Tensor]):
```

Ruff (0.12.2)

Line too long (138 > 120)

(E501)